docs/version-support-policy.md

```diff
@@ -0,0 +1,7 @@
+# Version Support Policy
+
+We have officially announced the L-2 version support policy for WooCommerce and WordPress core [since May 2021](https://developer.woocommerce.com/2021/05/12/woocommerce-payments-is-adopting-a-new-version-support-policy/).
+
+## What does this policy mean for contributors?
+
+As a contributor to WooCommerce Stripe Payment Gateway, you would be mindful when adding new features and functions to your PRs as your code needs to work within proper ranges of WordPress and WooCommerce versions. Although we have [a CI workflow](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/develop/.github/workflows/compatibility.yml) to check the compatibility between the Stripe extension and these ranges, it's still good for contributors to be aware of this policy.
```

Comment:
Would "Functionality" sound slightly more natural in this context?

Reply:
You mean instead of "functions", right? If so, I. have updated it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4031/commits/9bffa2fb998f62cf4bbf21095448a08bf5e233b5